pages/common/doctl-databases-db.md

```diff
@@ -0,0 +1,24 @@
+# doctl databases db
+
+> Manage specific databases that are served by a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db/>.
+
+- Run a doctl databases db command with an access token:
+
+`doctl databases db [command] --access-token {{ access_token }}`
+
+- Retrieve the name of the given database hosted in the given database cluster:
+
+`doctl databases db get {{ database id }} {{ db name }}`
```

Comment:
```suggestion
`doctl databases db get {{database_id}} {{db_name}}`
```
[Our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) mandates that placeholders_are_written_in_snake_case.